api/scripts/elastic_ops/pyproject.toml

```diff
@@ -6,13 +6,19 @@ authors = ["joel <joel@jataware.com>"]
 readme = "README.md"
 packages = [
   {include = "copy_es_docs"},
-  {include = "job_playground"}
+  {include = "job_playground"},
+  {include = "pop_didx_boxytas"}
 ]
 
 [tool.poetry.dependencies]
```

Comment:
The elastic_ops poetry directory contains various scripts of operations I've used to fix/debug/reproduce/backup problems or situations on dojo's `es`.